src/components/DataCard/DataCard.tsx

```diff
@@ -0,0 +1,213 @@
+    const themeMatch =
+      themeLookup[normalized] ||
+      themeLookup[normalized.replace(/-/g, "_")] ||
+      themeLookup[normalized.replace(/_/g, "-")] ||
+      themeLookup[normalizeKey(label)];
+
+    const finalLabel = themeMatch?.name ?? label;
+    const color = themeMatch?.color ?? "#018F39";
+
+    return {
+      key: `${normalized}-${index}`,
+      label: finalLabel,
+      color,
+      iconId: themeMatch ? macroThemes[themeMatch.id] : undefined,
+    };
+  });
+
+  const formattedDate = (() => {
+    if (!post.publication_date) return "-";
+    const parsed = new Date(post.publication_date);
+    if (Number.isNaN(parsed.getTime())) return post.publication_date;
+
+    return parsed.toLocaleDateString("pt-BR", {
+      day: "2-digit",
+      month: "2-digit",
+      year: "numeric",
+    });
+  })();
+
+  const sourceUrlFromId =
+    post.id && /^\d+$/.test(post.id)
+      ? `https://zenodo.org/records/${post.id}`
+      : "test"; // <-- change this later for proper redirect button workflow
+
+  const sourceUrl =
+    (post as any)?.recordUrl ??
+    (post as any)?.url ??
+    (post as any)?.source ??
+    sourceUrlFromId;
+
+  const handleDownload = async (url: string, name: string) => {
+    try {
+      const response = await fetch(url);
+      const blob = await response.blob();
+      const blobUrl = URL.createObjectURL(blob);
+      const link = document.createElement("a");
+      link.href = blobUrl;
+      link.download = name;
+      link.click();
+      URL.revokeObjectURL(blobUrl);
+    } catch (err) {
+      console.error("Erro ao baixar arquivo:", err);
+    }
+  };
+
+  return (
+    <div className="flex min-h-[176px] w-full flex-col gap-6 rounded-lg border border-[#EFEFEF] bg-[#F8F7F8] px-4 py-4 transition-transform duration-300 hover:scale-[1.01]">
+      <div className="flex flex-col gap-6 lg:flex-row lg:items-start lg:justify-between lg:gap-10">
+        <div className="flex flex-1 flex-col gap-4">
+          <div className="flex flex-col gap-3">
+            <h2 className="text-base font-semibold leading-4 text-[#292829]">
+              {post.title}
+            </h2>
+
+            {normalizedTags.length > 0 && (
+              <div className="flex flex-wrap gap-2">
+                {normalizedTags.map(({ key, label, color }) => (
+                  <span
+                    key={key}
+                    style={{
+                      backgroundColor: color ?? "#018F39",
+                      color: "#fff",
+                    }}
+                    className="min-h-[20px] min-w-[51px] rounded-full border border-[#E2E8F0] px-[10px] py-[2px] text-xs font-medium leading-4"
+                  >
+                    {label}
+                  </span>
+                ))}
+              </div>
+            )}
+          </div>
+
+          {post.description && (
+            <p className="text-base font-normal leading-[150%] text-[#292829]">
+              {post.description}
+            </p>
```

Comment:
Since we're receiving a string that's actually a HTML tag, we need to somehow parse it into HTML:


```suggestion
            <div dangerouslySetInnerHTML={{ __html: post.description }} className="text-sm"/>
```

:warning: This isn't the whole solution, because we need to maybe clean the HTML string we're given, so we can use this lib [DOMPurify](https://www.npmjs.com/package/dompurify) to help us with it